code-name-taran/src/com/spaghettisoft/component/scrabble/Checks.java

```diff
@@ -0,0 +1,181 @@
+package com.spaghettisoft.component.scrabble;
+
+import java.util.Scanner;
+
+public class Checks {

```

Comment:
The name of the class is too general. Maybe something like "OptionValidator" is better suited ^_^